config-files/usr/share/plasma/desktoptheme/openSUSE/lancelot/theme.config

```diff
@@ -0,0 +1,157 @@
+[Group-Default]
```

Comment:
AFAIK lancelot is dead, so this could be removed.

Reply:
Shall I remove the whole lancelot folder and its contents from all themes?

Reply:
Instead of adding files, yes. Keeping the old stuff around is IMO fine.